NASA-AMMOS/aerie-ui
# Feature/command dictionary

Resolves #1493 

To test:

1. Navigate to the `/dictionaries` page.
2. Upload the `/e2e-tests/data/command-dictionary.xml` or a command dictionary of your choice.
3. Navigate to `/parcels`
4. Create a new parcel with that command dictionary selected
5. Navigate to `/sequencing`
6. Create a new workspace
7. Create a new sequence using the parcel created in step 4
8. Verify the new `Command Dictionary` tab is visible in the right panel
9. Verify that clicking the tab displays a searchable listing all the commands in the dictionary
10. Click on a command
11. Verify that the arguments are displayed properly
12. Click on the "copy" icon at the top right of the panel
13. Verify that the command stem is successfully copied to the clipboard
14. Click on the "close" icon at the top right of the panel
15. Verify that the selected command is cleared
